Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -998,9 +983,9 @@ - (void)sidebarDidHide {
     self.tableView.allowsSelection = !(tagFilterType == SPTagFilterTypeDeleted);
     [self.tableView setBorderVisibile:NO];
     
-    addButton.customView.tintAdjustmentMode = UIViewTintAdjustmentModeAutomatic;
-    addButton.enabled = YES;
-    emptyTrashButton.enabled = (tagFilterType == SPTagFilterTypeDeleted && [self numNotes] > 0) || tagFilterType != SPTagFilterTypeDeleted ? YES : NO;
+    self.addButton.customView.tintAdjustmentMode = UIViewTintAdjustmentModeAutomatic;
+    self.addButton.enabled = YES;
+    self.emptyTrashButton.enabled = (tagFilterType == SPTagFilterTypeDeleted && [self numNotes] > 0) || tagFilterType != SPTagFilterTypeDeleted ? YES : NO;
```

Comment:
why do we need the ? here: ```tagFilterType != SPTagFilterTypeDeleted ? YES : NO;```

why not use just this: ```tagFilterType != SPTagFilterTypeDeleted ```

Reply:
Ahhhhhh I copied and pasted, and didn't notice that!! Patching up, thank you!!